abap-code-review/src/code_review.adoc

```diff
@@ -0,0 +1,62 @@
+== Best Practices
+
+This section contains suggestions on how to conduct code reviews effectively. Other code review guides have already dealt with this topic generically, for instance, link:https://google.github.io/eng-practices/[Google's Engineering Practices documentation]. We will focus on ABAP specific considerations.
+
+=== Why perform Code Reviews?
+
+Code reviews can detect issues in the software that are harder to find via other methods:
+
+* Foster link:https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md[Clean code] to improve *maintainability* and *readability* of code, including comments, and to avoid unnecessary *complexity*.
+* Ensure the code is *well designed*.
+* Consider behavioral aspects like *performance* and *security*.
+* Improve *testability* of code, including test design and adequateness of tests.
+
+Code reviews are one possibility to *share experience and knowledge* between developers. It works a bit like "asynchronous pair programming", so that colleagues can learn from each other, see different perspectives, and share know-how about all parts of the code.
+
+Code reviews help to prevent that *code health* deteriorates over time.
+
+=== Size and Speed of Reviews
+
+Change-based code reviews are these days considered more effective than formal link:https://en.wikipedia.org/wiki/Fagan_inspection[Fagan-style code inspections] as they were practiced some years ago. Conducted continuously, they require less effort while still finding a considerable number of defects at an earlier time.The main reason is the *smaller size* of code deltas to digest. This ensures that reviews can be performed both, more *quickly* and *thoroughly*, while keeping the process lean, i.e., less waste if changes are rejected and lower probability to block other development. Changes should be self-contained to help reviewers focus on the task at hand.
```

Comment:
Perhaps, you can use some advanced text editor like vim and reformat to 80-120 characters per line.
That helps pointing out problems in the text the same way as in source codes.

There is a missing space:
```
earlier time.The main
```

Reply:
thanks, added in https://github.com/SAP/styleguides/pull/183/commits/2cbdd63fbe659fc8d41f6799b847a1512b60ed69